.github/workflows/test.yaml

```diff
@@ -67,3 +67,28 @@ jobs:
       run: poetry install -n --all-extras --with test --with dev
     - name: Run mypy
       run: poetry run mypy --config-file mypy.ini pynamodb
+
+  flake8:
```

Comment:
We have flake8 running on run_qa.yml and test_group.yml. What is the difference between the three runs of flake8? Is there any redundancy?

Reply:
This is separate GA pipeline from `saturn-api`, so there is only one flake8 run here